R/trace_calls.R

```diff
@@ -42,11 +42,20 @@ trace_calls <- function (x, parent_functions = NULL, parent_ref = NULL) {
     }
   }
   else if (is.call(x)) {
-    src_ref <- attr(x, "srcref") %||% impute_srcref(x, parent_ref)
-    if ((identical(x[[1]], as.name("<-")) || identical(x[[1]], as.name("="))) && # nolint
-      (is.call(x[[3]]) && identical(x[[3]][[1]], as.name("function")))) {
-      parent_functions <- c(parent_functions, as.character(x[[2]]))
-    }
```

Comment:
Could you please re-indent this by just 2 spaces

Reply:
Done!